concent_api/core/transfer_operations.py

```diff
@@ -31,38 +31,33 @@
 logger = getLogger(__name__)
 
 
-def verify_file_status(client_public_key: bytes) -> None:
+def verify_file_status(
```

Comment:
Why splitting this? It has not more that 120 characters.

Reply:
probably IDE 'did it'.. Fixed :)